content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -121,16 +121,23 @@ allows the users to resize the volume by editing the corresponding PVC object.
 The following types of volumes support volume expansion, when the underlying
 Storage Class has the field `allowVolumeExpansion` set to true.
 
-* gcePersistentDisk
-* awsElasticBlockStore
-* Cinder
-* glusterfs
-* rbd
-* Azure File
-* Azure Disk
-* Portworx
-* FlexVolumes
-* CSI  {{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+{{< table caption = "Table of Volume types and the version of Kubernetes they require"  >}}
+
+Volume type | Required Kubernetes version
+:---------- | :--------------------------
+gcePersistentDisk | 1.11
+awsElasticBlockStore | 1.11
+Cinder | 1.11
+glusterfs | 1.11
+rbd | 1.11
+Azure File | 1.11
+Azure Disk | 1.11
+Portworx | 1.11
+FlexVolumes | 1.13
+CSI | 1.14 ,1.15-alpha ,1.16-beta
```

Comment:
```suggestion
CSI | 1.14 (alpha), 1.16 (beta)
```

Reply:
That looks much better. Thanks for the suggestion, @sftim . I will make the changes by EOD and resubmit for review

Reply:
Did you mean CSI | 1.14 ,1.15 (alpha) ,1.16 (beta) instead of CSI | 1.14 (alpha), 1.16 (beta) ?

Reply:
To me it's clear / obvious that something that goes alpha in v1.14 is and beta in v1.16 is also alpha in v1.15

Maybe that's not clear to all readers.

Reply:
Thank you for the clarification, Tim. I wasn't aware of that before. I have made the changes you suggested. If it all looks good, I will squash the commits into one.